### PR TITLE
fixes for OE master

### PR DESCRIPTION
**Pull request by pohly**
Fixes build errors encountered when updating to OE master. Tested as
part of https://github.com/ostroproject/ostro-os/pull/143

### CODE REVIEW

**Comment by pohly** [comment]
@tripzero and/or @rojkov: okay to merge?

File: meta-ostro/conf/distro/ostro.conf [hunk 1]
```diff
@@ -256,7 +256,23 @@ PREFERRED_RPROVIDER_java2-runtime = "openjdk-8"
 # and linux-yocto-edison 3.10.98 (IOTOS-1626, "fatal error: linux/compiler-gcc6.h: No such file or directory").
 #
 # Until these issues are fixed, we need to stay on gcc 5.3, the previous default.
```
Comment (rojkov) [inline meta-ostro/conf/distro/ostro.conf]: nitpick: the comment could mention just gcc 5.x

But looks ok to me

Comment (pohly) [inline meta-ostro/conf/distro/ostro.conf]: Agreed. I'll fix this before merging.

**Comment by pohly** [comment]
merged manually after fixing the nitpick

**Pull request closed without merging**